app/javascript/dashboard/components/ChatList.vue

```diff
@@ -313,12 +326,38 @@ export default {
         } else {
           conversationList = [...this.allChatList(filters)];
         }
+      } else if (this.hasAppliedFiltersOrActiveFolders) {
```

Comment:
Can you please add some comments about what exactly this block is doing?

Reply:
Yes, When we an in filters or any saved folder view and a new conversation is created that will show in the chat list even if the newly created conversation is not in that filter standard. By this, it will check when we are in filter or folders view it will show a banner with reload button instead of showing that conversation there. When we click the reload button it will call the filter API with page = 1 with the query of active filter or folder.

Reply:
Fixes by https://github.com/chatwoot/chatwoot/pull/3922#discussion_r800402728